docs/tutorials/uniq-factories/uniq-offer/uniq-offer.experimental.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Uniq Offer'
+
+order: 1
+---
+
+# Uniq Offer
+
+## Overview of Uniq Offer feature
+
+Uniq Offer allow user to make a buy offer directly to the owner of Uniq or Uniq Factory if it's tradable.
```

Comment:
Should be `Uniq Offer allows a user to make a buy offer ...`.

Reply:
fixed